docs/integrations/notification/index.rst

```diff
@@ -0,0 +1,126 @@
+#####################################
+ Monitor Experiment Through Webhooks
+#####################################
+
+This section includes user guides for how to set up notifications for experiments.
```

Comment:
This is essentially restating the title. Writing a good intro is tough so I just came up with something for you here:

> Monitoring experiment status is a vital part of working with Determined. In order to integrate Determined into your existing workflows, you can make use of webhooks to update other systems, receive emails, slack messages, and more when an experiment is updated.

Reply:
Thank you~